example/hands-on/config/test_db_config.yml

```diff
@@ -0,0 +1,8 @@
+:test_db:
```

Comment:
why do you commit this file?

Reply:
I commit all files which are created in this example.